src/UILayer/CQRSStoreGroup.ts

```diff
@@ -14,6 +14,36 @@ import { ChangedPayload } from "../payload/ChangedPayload";
 import { Dispatcher } from "../Dispatcher";
 const CHANGE_STORE_GROUP = "CHANGE_STORE_GROUP";
 
+
+/**
+ * This function is type converter for TypeScript.
+ * It has actual implementation, but it is only use for typing.
+ *
+ * - Status: unstable
+ * 
+ * This is useful for TypeScript, but it is unstable API.
+ * You can use it own risk.
+ * 
+ * ## Example
+ *
+ * ```ts
+ * const mapping = {
+ *   a: new AStore(),
+ *   b: new BStore()
+ * };
+ * const StoreState = MapStoreToState(mapping);
+ * // Get Mapped to State for Store
+ * type States = typeof StoreState
+ * // {
+ * //   a: AState,
+ * //   b: BState
+ * // };
+ * ```
+ */
+export function MapStoreToState<T>(_mapping: MapStore<T>): MapState<T> {
+    return {} as MapState<T>
+}
```

Comment:
Hmm, I think in general you first declare `T` and share it for typing `MapStore` and `MapState`, though it may feel like a bit verbose.

```ts
type States = {
    aState: AState;
    bState: BState;
};
const mapping: MapStore<States> = {
     aState: new AStore(),
     bState: new BStore()
};

const storeGroup = new CQRSStoreGroup(mapping);

// ...

const state = context.getState<MapState<States>>();
```

Though this is a breaking change, I wonder if you can make the `Context` class itself generic instead of `getState()`, i.e.

```ts
class Context<T> {
  // you also have to make StoreLike generic
  constructor({dispatcher, store}: {dispatcher: Dispatcher; store: StoreLike<T>;}) {
    // ...
  }
  getState(): MapState<T> {
    // ...
  }
}
```

This way you don't need to explicitly specify the type parameter as TypeScript can infer it correctly.
This is actually much more type safe than `getState<T>(): T`, whose return type is virtually `any`.